programmers/20210510/음양-더하기-for.js

```diff
@@ -0,0 +1,8 @@
+function solution(absolutes, signs) {
+    let sum = 0;
+    for(let i = 0; i < signs.length; i++) {
+        if (signs[i] === true) sum += absolutes[i];
```

Comment:
=== true를 쓰지 않는다면 더 가독성 좋은 코드가 될꺼 같아요
```suggestion
        if (signs[i]) sum += absolutes[i];
```